.github/ISSUE_TEMPLATE/avm_module_issue.yml

```diff
@@ -48,6 +48,7 @@ body:
         # - "avm/ptn/avd-lza/networking"
         # - "avm/ptn/avd-lza/session-hosts"
         - "avm/ptn/azd/acr-container-app"
+        - "avm/ptn/azd/aks"
```

Comment:
Thanks @jerryshia, the PR is good to go. Something went wrong with the resolution of conflicts for the ML one though, we we have to wait until this PR is merged (and may need to resolve conflicts again): https://github.com/Azure/bicep-registry-modules/pull/3511
Otherwise, the static tests in upstream will prevent this module from being published. Will let you know once it is merged.

Reply:
Ok, no conflict in sight. Approving